src/components/application_manager/rpc_plugins/app_service_rpc_plugin/include/app_service_rpc_plugin/commands/mobile/get_app_service_data_request.h

```diff
@@ -67,6 +67,11 @@ class GetAppServiceDataRequest : public app_mngr::commands::CommandRequestImpl {
    **/
   virtual void Run();
 
+  /*
```

Comment:
@YarikMamykin Fix Doxygen comment

Reply:
@AByzhynar fixed in 7fac4d2c0b4cd1d6ce1768602598fa10b9cf6c14